cmd/trtl/main.go

```diff
@@ -530,6 +546,195 @@ func initPeersClient(c *cli.Context) (err error) {
 // Trtl (DB) Client Functions
 //===========================================================================
 
+// migrateCerts creates certificate records in the trtl database based on the existing certreqs.
+func migrateCerts(c *cli.Context) (err error) {
+	ctx, cancel := profile.Context()
+	defer cancel()
+
+	// Open the report file if requested
+	var report *os.File
+	if c.String("report") != "" {
+		if report, err = os.Create(c.String("report")); err != nil {
+			return cli.Exit(fmt.Errorf("could not open report file %s: %s", c.String("report"), err), 1)
+		}
+		defer report.Close()
+	}
+
+	// Get the list of certreqs
+	req := &pb.CursorRequest{
+		Namespace: wire.NamespaceVASPs,
+	}
+	var stream pb.Trtl_CursorClient
+	if stream, err = dbClient.Cursor(ctx, req); err != nil {
+		return cli.Exit(err, 1)
+	}
+
+	var migrated, skipped int
+	var migrateErr error
+
+	for {
+		var rep *pb.KVPair
+		if rep, err = stream.Recv(); err != nil {
+			if err != io.EOF {
+				migrateErr = err
+			}
+			break
+		}
+
+		vasp := &gds.VASP{}
+		if err = proto.Unmarshal(rep.Value, vasp); err != nil {
+			migrateErr = err
+			break
+		}
+
+		// Get all the certificate requests for this vasp
+		var ids []string
+		if ids, err = models.GetCertReqIDs(vasp); err != nil {
+			migrateErr = err
+			break
+		}
+
+		// Get the latest completed certificate request
```

Comment:
I guess since we overwrite the identity certificate, we won't be able to capture the old certificates from the older certificate requests.

Reply:
Yeh, that's unfortunate -- there are only a handful through, so I could manually create those entries if we do this migration soon - mind creating a follow on story for that?